apps/site/test/site_web/views/schedule_view_test.exs

```diff
@@ -1025,7 +1025,7 @@ defmodule SiteWeb.ScheduleViewTest do
   end
 
   describe "route_header_tabs/1" do
-    test "returns 5 tabs for commuter rail (1 hidden by css)", %{conn: conn} do
+    test "returns 5 tabs for commuter rail (2 hidden by css)", %{conn: conn} do
```

Comment:
I think we should change this function under test here to not return the schedule tab at all for commuter rail, it doesn't look like it would be difficult: https://github.com/mbta/dotcom/blob/master/apps/site/lib/site_web/views/schedule_view.ex#L386 

One was hidden by css before in this test because one was only shown on mobile to allow for better responsiveness. We had to adjust the size of the tabs on mobile so that 4 tabs would fit across.